src/main/java/com/rackspacecloud/metrics/ingestionservice/influxdb/InfluxDBHelper.java

```diff
@@ -177,34 +177,25 @@ public InfluxDBFactory getInfluxDBFactory() {
      * @param tenantId
      * @param measurement
      * @return
-     * @throws Exception
      */
-    private TenantRoutes getTenantRoutes(String tenantId, String measurement) throws Exception {
+    private TenantRoutes getTenantRoutes(String tenantId, String measurement) {
         TenantRoutes tenantRoutes;
 
-        try {
-            tenantRoutes = routeProvider.getRoute(tenantId, measurement, restTemplate);
-        }
-        catch(Exception e) {
```

Comment:
I'm wondering if other runtime exceptions were likely here in which case they'll get bubbled up as-is...which could be fine.

Reply:
Ahhh yeah one of the implementing classes throws a RestClientException on this function call.